src/main/java/org/broadinstitute/hellbender/tools/picard/illumina/parser/ClusterData.java

```diff
@@ -78,23 +61,6 @@ public void setLane(final int lane) {
         this.lane = lane;
     }
 
-    public boolean laneIsSet() {
-        return lane != -1;
-    }
-
-    /**
-     * Either set this value if not already set, or if already set, throw an exception if new value != current value.
-     */
-    public void setOrCheckLane(final int lane) {
-        if (laneIsSet()) {
-            if (this.lane != lane) {
-                throw new IlluminaParserException("Lane number mismatch for " + this + " : " + this.lane + " != " + lane);
-            }
-        } else {
-            this.lane = lane;
-        }
-    }
```

Comment:
these methods were dead - they have no semantics.
anyway, moot point. we're deleting this class.